source/server/infrastructure/snapshotter.coffee

```diff
@@ -1,27 +1,52 @@
 class Space.eventSourcing.Snapshotter extends Space.Object
 
-  _collection: null
+  @snapshotsCollection: null
+
+  Dependencies: {
+    configuration: 'Configuration'
+    ejson: 'EJSON'
+    injector: 'Injector'
+    mongo: 'Mongo'
+  }
+
+  collection: null
   _versionFrequency: 0
 
-  constructor: (config) ->
-    @_collection = config.collection
-    @_versionFrequency = config.versionFrequency
+  onDependenciesReady: ->
+    @_setupSnapshotting() if @configuration.eventSourcing.snapshotting.enabled
```

Comment:
Yep. That property would be better called `bootstrapSingletons` or something similar